_topic_map.yml

```diff
@@ -553,8 +553,8 @@ Topics:
   File: managing_networking
 - Name: Configuring Service Accounts
   File: service_accounts
-- Name: Managing Authorization Policies
-  File: manage_authorization_policy
+- Name: Managing Role-based Access Control
+  File: manage_rbac
```

Comment:
@ahardin-rh - will this change require a redirect?

Reply:
@vikram-redhat I think so since the file is now renamed. Just in case someone has this page bookmarked as the old file name. I updated all file mentions in the docs.

Reply:
Thanks. I have added it to the redirects register. Since this is only occurring from 3.7 onwards, I have set this up to redirect the 'latest' and not the 3.6 version.